frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,21 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  className?: string;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-  <button type={type}>{label}</button>
+const Button: React.FC<Properties> = ({
+  type = 'button',
+  label,
+  className = '',
+}) => (
+  <button type={type} className={getValidClassNames(className, styles['btn'])}>
```

Comment:
className should override styles['btn'], so swap them

Reply:
done